localgov_microsites_base.theme

```diff
@@ -686,3 +688,19 @@ function localgov_microsites_base_theme_suggestions_page_alter(array &$suggestio
     }
   }
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function localgov_microsites_base_preprocess_field(&$variables) {
+  $element = $variables['element'];
+
+  // Load the group by context. See localgov_microsites_group.module.
+  $group = localgov_microsites_group_get_by_context();
```

Comment:
If we use this we'll need to lock the localgov_microsites_base package in the 3.x localgov_microsites profile branch to prevent issues similar to https://github.com/localgovdrupal/localgov_microsites/pull/494